bolt-taproot-gossip.md

```diff
@@ -0,0 +1,748 @@
+  * [Timestamp fields](#timestamp-fields)
+  * [Bootstrapping Taproot Gossip](#bootstrapping-taproot-gossip)
+  * [Specification](#specification)
+    * [Feature Bits](#feature-bits)
+    * [The `announcement_signatures_2` message](#the-announcement_signatures_2-message)
+    * [The `channel_announcement_2` message](#the-channel_announcement_2-message)
+    * [The `node_announcement_2` message](#the-node_announcement_2-message)
+    * [The `channel_update_2` message](#the-channel_update_2-message)
+  * [Appendix A: Algorithms](#appendix-a-algorithms)
+      * [Partial Signature Calculation](#partial-signature-calculation)
+      * [Partial Signature Verification](#partial-signature-verification)
+  * [Appendix B: Test Vectors](#appendix-b-test-vectors)
+  * [Acknowledgements](#acknowledgements)
+
+## Aim 
+
+This document aims to update the gossip protocol defined in [BOLT 7][bolt-7] to
+allow for advertisement and verification of taproot channels. An entirely new
+set of gossip messages are defined that use Schnorr signatures and that use a
+purely TLV based schema.
+
+## Overview
+
+The initial version of the Lightning Network gossip protocol as defined in
+[BOLT 7][bolt-7] was designed around P2WSH funding transactions. For these
+channels, the `channel_announcement` message is used to advertise the channel to
+the rest of the network. Nodes in the network use the content of this message to
+prove that the channel is sufficiently bound to the Lightning Network context
+and that it is owned by the nodes advertising the channel. This proof and
+verification protocol is, however, not compatible with SegWit V1 (P2TR)
+outputs and so cannot be used to advertise the channels defined in the
+[Simple Taproot Channel][simple-taproot-chans] proposal. This document thus aims
+to define an updated gossip protocol that will allow nodes to both advertise and
+verify taproot channels. This part of the update affects the
+`announcement_signatures` and `channel_announcement` messages.
+
+The opportunity is also taken to rework the `node_announcement` and
+`channel_update` messages to take advantage of Schnorr signatures and TLV
+fields. Timestamp fields are also updated to be block heights instead of Unix
+timestamps.
+
+## Terminology
+
+- Collectively, the set of new gossip messages will be referred to as 
+  `taproot gossip`
+- `node_1` and `node_2` refer to the two parties involved in opening a channel.
+- `node_ID_1` and `node_ID_2` respectively refer to the public keys that
+  `node_1` and `node_2` use to identify their nodes in the network.
+- `bitcoin_key_1` and `bitcoin_key_2` respectively refer to the public keys
+  that `node_1` and `node_2` will use for the specific channel being opened.
+  The funding transactions output will be derived from these two keys.
+
+## Type Definitions
+
+The following convenient types are defined:
+
+* `schnorr_signature`: a 64-byte Schnorr signature as defined
+  in [BIP340][bip-340].
```

Comment:
https://github.com/lightning/bolts/pull/798/commits/2f5d00bd4b6d2b495b0a0910debba751c71d3d3d used "bip340sig".

Personally, as Schnorr chose to patent, I would prefer to refer to them by patent number (though TBH a quick google failed to find it).  But maybe that's just me?